packages/preact-query-persist-client/src/PersistQueryClientProvider.tsx

```
@@ -0,0 +1,55 @@
+import { useEffect, useRef, useState } from 'preact/hooks'
+import type { VNode } from 'preact'
```

ESLint

[error] `preact` type import should occur after import of `@tanstack/preact-query`

(import/order)